core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -573,6 +573,19 @@
  * <td>Distribution Summary</td>
  * <td></td>
  * </tr>
+ * <!-- Balancing -->
+ * <tr>
+ * <td>N/A</td>
+ * <td>N/A</td>
+ * <td>{@value METRICS_MANAGER_BALANCER_MIGRATIONS_IN_PROGRESS}</td>
+ * <td>Gauge</td>
+ * </tr>
+ * <tr>
+ * <td>N/A</td>
+ * <td>N/A</td>
+ * <td>{@value METRICS_MANAGER_BALANCER_MIGRATIONS_NEEDED}</td>
+ * <td>Gauge</td>
```

Comment:
I don't understand what this metric represents.  Some notes here would be useful

Reply:
Added some note details and removed the in_progress metric value